moveit_ros/hybrid_planning/test/config/global_planner.yaml

```diff
@@ -6,8 +6,10 @@ planning_scene_monitor_options:
   robot_description: "robot_description"
   joint_state_topic: "/joint_states"
   attached_collision_object_topic: "/moveit_cpp/planning_scene_monitor"
-  publish_planning_scene_topic: "/moveit_cpp/publish_planning_scene"
-  monitored_planning_scene_topic: "/moveit_cpp/monitored_planning_scene"
+  # Subscribe to this topic (The name comes from the perspective of moveit_cpp)
+  publish_planning_scene_topic: "/planning_scene"
+  # Publish this topic, e.g. to visualize with RViz
+  monitored_planning_scene_topic: "/global_planner/planning_scene"
```

Comment:
The naming of these parameters is atrocious from the perspective of the hybrid planner. I'm not sure what can be done about it since the parameters get passed to moveit_cpp.

Reply:
I think it would make sense to use the MoveItCpp options directly and load them with a moveit_cpp.yaml. But you are right, those parameters are confusing

Reply:
Moving the parameters to a separate `moveit_cpp.yaml` makes a lot of sense as well as reducing LOC. Done.

Yep, those parameters will still look confusing if somebody does `ros2 param list`.

Reply:
Actually I'm not sure that will work. I have to test if the parameters are read properly or if it just uses defaults.

Reply:
@henningkayser I tried this in commit [d2cf1b3](https://github.com/ros-planning/moveit2/pull/1052/commits/d2cf1b321fcafbba85208730c9c3824104f3c9f7). You can quickly verify that it doesn't work:

- Modify one of the topics in global_planner_moveit_cpp.yaml, for example, like this:  `monitored_planning_scene_topic: "test/global_planner/planning_scene"`
- Launch the hybrid planning demo:  `ros2 launch moveit_hybrid_planning hybrid_planning_demo.launch.py`
- List the subscriptions of the global planner:  `ros2 node info /global_planner_private_`

You will see that it does not subscribe to the right topic.

I'm going to revert that commit

Reply:
I'm going to try to write some small tests to try to understand how declaring and overriding parameters work.  I think the ros2 interface is confusing and I don't know what the right answer is.

Reply:
Can we get this merged and make an issue in the meantime? It's better than the previous state because at least these parameters are being used.